docker-compose.yaml

```diff
@@ -0,0 +1,35 @@
+version: "3.4"
+
+services:
+  production:
+    build: 
+      context: .
+      target: production
+      dockerfile: server/Dockerfile
+    ports: ["8000:80"]
```

Comment:
This runs the production build and serves it on port 80, exposed as 8000 on the host.